test/perceptual/alert.html

```diff
@@ -0,0 +1,39 @@
+
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="utf-8">
+		<title>Alert Snapshot</title>
+		<meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1, user-scalable=yes">
+		<script src="https://s.brightspace.com/lib/webcomponentsjs/0.7.21/webcomponents-lite.min.js"></script>
+		<link rel="import" href="alert.html.ignore"> 
+		<style>
+			@import 'https://fonts.googleapis.com/css?family=Lato:700,400,300';
+			html {
+				font-size: 20px;
+			}
+			body {
+				font-family: 'Lato', sans-serif;
+			}
+		</style>
+	</head>
+	<body>
+		<p class="screenshot" style="display:none;">
+			<vui-alert>Oops! This is embarassing...</vui-alert>
+		</p>
+		<p id="loading">Loading</p>
```

Comment:
Some of this can be cleaned up.  I think Dave L. added this in button as he was debugging some of the perceptual diff issues we are seeing.